tools/packages/minion/minion.spec

```diff
@@ -167,11 +167,27 @@ mv "%{buildroot}%{minioninstprefix}/etc/minion.conf" "%{buildroot}%{_sysconfdir}
 # container package files
 find %{buildroot}%{minioninstprefix} ! -type d | \
     grep -v %{minioninstprefix}/bin | \
+    grep -v %{minioninstprefix}/etc | \
     grep -v %{minionrepoprefix} | \
-    grep -v %{minioninstprefix}/etc/featuresBoot.d | \
-    grep -v %{minioninstprefix}/etc/org.opennms.minion.controller.cfg | \
```

Comment:
We should keep the user's `org.opennms.minion.controller.cfg` too.

Reply:
We do, it's specified down in the %files section

https://github.com/OpenNMS/opennms/blob/b3bf1dcda7565e63e1ac775b705941e36cfe2c76/tools/packages/minion/minion.spec#L251